src/code/RepositorySettings.cs

```diff
@@ -168,28 +168,27 @@ public static PSRepositoryInfo Update(string repoName, Uri repoUri, int repoPrio
 
                 // determine if existing repository node (which we wish to update) had Url to Uri attribute
```

Comment:
nitpick, I think I left this typo in last time
```suggestion
                // determine if existing repository node (which we wish to update) had Url or Uri attribute
```